ipad/ViewControllers/Shift/ShiftViewController.swift

```diff
@@ -177,12 +193,14 @@ class ShiftViewController: BaseViewController {
     @objc func completeAction(sender: UIBarButtonItem) {
         guard let model = self.model else { return }
         self.dismissKeyboard()
+
+        self.updateStatuses()
```

Comment:
Added updateStatus before checking canSubmit, so we have the most up to date info on the validation of inspections